charts/back8sup/values.yaml

```diff
@@ -12,7 +12,7 @@ image:
   # image.pullPolicy -- set the image pullPolicy
   pullPolicy: IfNotPresent
   # image.tag -- set the tag of the image
-  tag: "latest"
+  tag: "v0.1.3"
```

Comment:
Did you consider using just the appVersion like other charts do?

Reply:
see https://github.com/adfinis-sygroup/helm-charts/pull/43/commits/9daa637056e34622e14c12e40c2741b5cd8635b9